index.js

```diff
@@ -8,3 +8,121 @@ function closeNav() {
+for (let i = 0; i < projects.length; i += 1) {
+  const project = document.createElement('div');
+  project.className = 'card';
+  project.innerHTML = `
+  <img src="${projects[i].img}" alt="Snapshoot-portfolio"/>
+<div class="info-block">
+  <div class="client">
+    <h2 class="project-title">${projects[i].title}</h2>
+    <div class="details">
+      <div class="name"><p class="project-name">${projects[i].client}</p></div>
+      <div class="circle-1"></div>
+      <div class="role"><p class="project-role">${projects[i].role}</p></div>
+      <div class="circle-2"></div>
+      <div class="year"><p class="project-year">${projects[i].year}</p></div>
+    </div>
+  </div>
+  <p class="summary">
+  ${projects[i].desc}
+  </p>
+  <div class="languages">
+    <div class="html"><p class="project-html">${projects[i].tech[0]}</p></div>
+    <div class="css"><p class="project-css">${projects[i].tech[1]}</p></div>
+    <div class="js"><p class="project-js">${projects[i].tech[2]}</p></div>
+  </div>
+  <div class="action">
+    <button class="see_project"><span class="see">${projects[i].button}</span></button>
+  </div>
+</div>`;
+  const button = project.querySelector('.see_project');
+  button.addEventListener('click', () => {
+    document.querySelector('.popup').style.display = 'flex';
+  });
+  projectsContainer.append(project);
+}
+for (let i = 0; i < projects.length; i += 1) {
+  const popup = document.createElement('div');
+  popup.className = 'popup';
+  popup.innerHTML = `
+  <img src="${projects[i].img}" alt="Snapshoot-portfolio"/>
+<div class="popup-info-block">
+  <div class="popup-client">
+    <h2 class="popup-project-title">${projects[i].title}</h2>
+    <div class="popup-details">
+      <div class="popup-name"><p class="popup-project-name">${projects[i].client}</p></div>
+      <div class="popup-circle-1"></div>
+      <div class="popup-role"><p class="popup-project-role">${projects[i].role}</p></div>
+      <div class="popup-circle-2"></div>
+      <div class="popup-year"><p class="popup-project-year">${projects[i].year}</p></div>
+    </div>
+  </div>
+  <p class="popup-summary">
+  ${projects[i].desc}
+  </p>
+  <div class="popup-languages">
+    <div class="popup-html"><p class="popup-project-html">${projects[i].tech[0]}</p></div>
+    <div class="popup-css"><p class="popup-project-css">${projects[i].tech[1]}</p></div>
+    <div class="popup-js"><p class="popup-project-js">${projects[i].tech[2]}</p></div>
+  </div>
+  <button id="go-live"><img src="assets/images/source.jpg" alt="github">Go live</button>
+  <button id="source"><img src="assets/images/images.png" alt="github">See Source</button>
+  <a href="javascript:void(0)" id="close-btn-n" class="closebtn-n">&times;</a>
+</div>`;
+  const button = popup.querySelector('#close-btn-n');
+  button.addEventListener('click', () => {
+    document.querySelector('.popup').style.display = 'none';
+  });
+  projectsContainer.append(popup);
```

Comment:
- [ ] By running a for loop you are creating extra code for no reason. Also, the function to display the popup, currently, displays the popup of only the first project for any project that the user clicks. The popup must show the content of the project that the user clicks on. It would be easier to create the popup when the user clicks and remove the HTML code after the user clicks on the 'X' button. Kindly fix the bug that makes only the first project popup displayed.
 - [ ] The popup on the mobile view works but the styling is quite different from the Figma template. Kindly add some margins and a dark gray overlay like the template to make your popup looks like it is floating above the content below, reduce the whitespace in the center, change the image, title, and title details order, and work on the style of the buttons to make it look like the Figma template(colors, sizing, hover effect, etc).
 - Your mobile view:
 - 
<img width="451" alt="image" src="https://user-images.githubusercontent.com/61919617/217374480-a4dd2f2b-e2fe-4a47-bfcb-359be115b8b1.png">

 - Figma template mobile:
 - 
<img width="269" alt="image" src="https://user-images.githubusercontent.com/61919617/217374545-b41cc851-b7ca-4dc3-900b-b029ff69c545.png">

 - [ ] Currently, your popup on the desktop view looks exactly the same as the mobile view. Please add the styling for the desktop view as the Figma desktop popup. Kindly take care of all the issues I mentioned on the mobile view of your popup window.
 - Your desktop view:
 - 
<img width="897" alt="image" src="https://user-images.githubusercontent.com/61919617/217375045-12a099d5-0ad6-462c-8024-f6011c2d904c.png">

 - Figma desktop view:
 - 
<img width="627" alt="image" src="https://user-images.githubusercontent.com/61919617/217375126-b5bd4778-df9b-47e3-be85-a7bce80b741e.png">

Reply:
I will be grateful if you told how to make the button create the content. or to give me a reference to learn from.